db/migrations/20220920084726_create_reviewImages_tables.sql

```diff
@@ -0,0 +1,10 @@
+-- migrate:up
+CREATE TABLE reviewImages(
```

Comment:
```suggestion
CREATE TABLE review_images(
```
테이블 네이밍 컨벤션을 지켜주세요-!